src/compositions/level-config.ts

```diff
@@ -0,0 +1,88 @@
+
+import {
+  MAP_ICON_IMG,
+  MAP_ICON_SPECIAL_IMG,
+  MAP_LOCK_IMG,
+  STAR_IMG,
+  SPECIAL_LEVELS,
+} from "@constants";
+import { syncLoadingImages } from '@common';
+
+const isSpecialLevel = (index) => SPECIAL_LEVELS.includes(index);
+
+export const getdefaultCloudBtnsPos = (canvas) => {
+    return [
+      [
+        [
+            canvas.width / 10,
+            canvas.height / 10
+        ],
+        [
+            canvas.width / 2.5,
+            canvas.height / 10
+        ],
+        [
+          canvas.width / 3 + canvas.width / 2.8,
+          canvas.height / 10,
+        ],
+        [
+            canvas.width / 10,
+            canvas.height / 3
+        ],
+        [
+            canvas.width / 2.5,
+            canvas.height / 3
+        ],
+        [
+            canvas.width / 3 + canvas.width / 2.8,
+            canvas.height / 3,
+        ],
+        [
+            canvas.width / 10,
+            canvas.height / 1.8
+        ],
+        [
+            canvas.width / 2.5,
+            canvas.height / 1.8
+        ],
+        [
+          canvas.width / 3 + canvas.width / 2.8,
+          canvas.height / 1.8,
+        ],
+        [
+            canvas.width / 2.5,
+            canvas.height / 1.3
+        ],
+      ],
+    ];
+}
+
+export const loadLevelImages = async () => {
+    return await syncLoadingImages(
```

Comment:
@bernhardccodev i think theres no need to await here since the caller can handle the Promise.